packages/apps/human-app/server/src/modules/job-assignment/job-assignment.mapper.ts

```diff
@@ -18,10 +26,50 @@ export class JobAssignmentProfile extends AutomapperProfile {
 
   override get profile() {
     return (mapper: Mapper) => {
-      createMap(mapper, JobAssignmentDto, JobAssignmentCommand);
-      createMap(mapper, JobAssignmentCommand, JobAssignmentData);
-      createMap(mapper, JobsFetchParamsDto, JobsFetchParamsCommand);
-      createMap(mapper, JobsFetchParamsCommand, JobsFetchParamsData);
+      createMap(
+        mapper,
+        JobAssignmentDto,
+        JobAssignmentParams,
+        namingConventions({
+          source: new SnakeCaseNamingConvention(),
+          destination: new CamelCaseNamingConvention(),
+        }),
+      );
+      createMap(
+        mapper,
+        JobAssignmentDto,
+        JobAssignmentCommand,
+        forMember(
+          (destination) => destination.data,
+          mapWith(JobAssignmentParams, JobAssignmentDto, (source) => source),
+        ),
+        namingConventions({
+          source: new SnakeCaseNamingConvention(),
+          destination: new CamelCaseNamingConvention(),
+        }),
+      );
+      createMap(
+        mapper,
+        JobsFetchParamsDto,
+        JobsFetchParams,
+        namingConventions({
+          source: new SnakeCaseNamingConvention(),
+          destination: new CamelCaseNamingConvention(),
+        }),
+      );
+      createMap(
+        mapper,
+        JobsFetchParamsDto,
+        JobsFetchParamsCommand,
+        forMember(
+          (destination) => destination.data,
+          mapWith(JobsFetchParams, JobsFetchParamsDto, (source) => source),
+        ),
+        namingConventions({
+          source: new SnakeCaseNamingConvention(),
```

Comment:
FYI: this cannot be extracted to the const, nor to function: automapper has then problems with correctly defining mapped types